### PR TITLE
Added more tests, merged master.

### PR DESCRIPTION
**Pull request by dvirk**

### CODE REVIEW

File: model/graph.py [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+    Represents an object within the game world.
+
+    As components it should have a description, as well as a collection of relationships (edges) to other nodes. The
+    string method should spit the node’s description and its relationships. There should a method that outputs all nodes
+    that the given nodes has a `contains’ relationship with, as well as the node it’s `contained in’.
+
+    """
+
+    def __init__(self, description=None, edge_list=None):
         self.description = None
```
Comment (mick-warehime) [inline model/graph.py]: _desc?

File: model/graph.py [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
         self.edge_list = None
```
Comment (mick-warehime) [inline model/graph.py]: _edge

File: model/graph.py [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
 
 
 class Edge(object):
+    CAT_PARENT_CHILD = 0
```
Comment (mick-warehime) [inline model/graph.py]: use enum / don't abbreviate category

File: model/graph_test.py [hunk 4]
```diff
@@ -16,3 +16,13 @@ def test_edge_is_directed_default_false(self):
         edge = graph.Edge(None, None, None)
 
         self.assertTrue(not edge.is_directed(), "Expected False, got %s" % str(edge.is_directed()))
+
+    def test_edge_returns_nodes(self):
+        edge = graph.Edge(1, 2, None)
```
Comment (mick-warehime) [inline model/graph_test.py]: use actual nodes?

File: model/graph.py (hunk 3 above)
Comment (dvirk) [inline model/graph.py]: Done

File: model/graph.py (hunk 2 above)
Comment (dvirk) [inline model/graph.py]: Done.

File: model/graph_test.py (hunk 4 above)
Comment (dvirk) [inline model/graph_test.py]: Done.

File: model/graph.py (hunk 1 above)
Comment (dvirk) [inline model/graph.py]: Done.

**Pull request merged**